tf_c.sh

```diff
@@ -44,6 +48,16 @@ else
 
     trap unload SIGINT
 
+    #check if log file exists
+    ls /tmp/tf_c.log > /dev/null
+    if [ $? -ne 0 ]; then
+	sudo --user=#1000 touch /tmp/tf_c.log #creates the log file as a user
+	if [ $? -ne 0 ]; then
+	    echo "Could not create log file"
+	    exit 1
+	fi
+    fi
+
```

Comment:
I don't think this is necessary. `fopen` will catch this error and create the file if it doesn't exist.